tictactoe.py

```diff
@@ -43,75 +43,126 @@
 # # if userX = "11":
 
 
-while True:
-    DEFAULTSPACE = "[ ]"
-    XGAMEPIECE = " X "
-    OGAMEPIECE = " O "
-    WINMESSAGE = "\n        Victory!"
-
-    gameBoard = []
+DEFAULTSPACE = "[ ]"
+XGAMEPIECE = " X "
+OGAMEPIECE = " O "
+WINMESSAGE = "\n        Victory!"
+DRAWMESSAGE = "\n          Draw!"
+
+
+gameBoard = []
+
+for _ in range(9):
+    gameBoard.append(DEFAULTSPACE)
+
+
+def displayBoard():
+    print("\n", gameBoard[0:3], "     ", [1, 2, 3],
+          "\n", gameBoard[3:6], "     ", [4, 5, 6],
+          "\n", gameBoard[6:10], "     ", [7, 8, 9],
+          "\n")
+
+
+def checkPlay(userMove, userGamePiece):
+    if userMove == ("1"):
+        gameBoard[0] = userGamePiece
+    if userMove == ("2"):
+        gameBoard[1] = userGamePiece
+    if userMove == ("3"):
+        gameBoard[2] = userGamePiece
+    if userMove == ("4"):
+        gameBoard[3] = userGamePiece
+    if userMove == ("5"):
+        gameBoard[4] = userGamePiece
+    if userMove == ("6"):
+        gameBoard[5] = userGamePiece
+    if userMove == ("7"):
+        gameBoard[6] = userGamePiece
+    if userMove == ("8"):
+        gameBoard[7] = userGamePiece
+    if userMove == ("9"):
+        gameBoard[8] = userGamePiece
+
+
+def checkWin(gameBoard):
```

Comment:
it might be nice if it tells you what player has won. You could add another if in the win scenario that checks if the matching characters are X or O